mmv1/products/identityplatform/Tenant.yaml

```diff
@@ -63,3 +64,23 @@ properties:
       Whether authentication is disabled for the tenant. If true, the users under
       the disabled tenant are not allowed to sign-in. Admins of the disabled tenant
       are not able to manage its users.
+  - name: 'client'
+    type: NestedObject
+    description: |
+      Options related to how clients making requests on behalf of a tenant should be configured.
+    properties:
+      - name: 'permissions'
+        type: NestedObject
+        description: |
+          Configuration related to restricting a user's ability to affect their account.
+        properties:
+          - name: 'disabledUserSignup'
+            type: Boolean
+            description: |
+              When true, end users cannot sign up for a new account on the associated project through any of our API methods.
+            default_from_api: true
```

Comment:
I'm slightly curious if you actually need this setting for both of these fields. Did you run into behavior the required them to default from the API?